ThreeBotPackages/threebot/farmmanagement/frontend/components/nodeinfo/nodeinfo.js

```diff
@@ -16,9 +17,22 @@ module.exports = new Promise(async (resolve, reject) => {
         this.node.free_to_use = false;
       }
     },
+    computed: {
+      cpr_total: function () {
+        const cu = this.ru2cu(this.node.totalResources)
+        return 1.5 * cu['cu'] + cu['su']
```

Comment:
can be written as: `return 1.5 * cu.cu + cu.su`

Also I suggest you move the ru2cu method inside this function, since this is not used anywhere else